public/form/css/styles.css

```diff
@@ -117,7 +117,7 @@ header {
   max-width: 175px;
   margin: 0;
   text-indent: -10000px;
-  background: url("../img/logo-mobile@2x.png") no-repeat center center;
+  background: url("../img/gfw_logo_hor.svg") no-repeat center center;
```

Comment:
please make sure the logo is the same size as in the main site - right now is smaller